### ISSUE TITLE
In constructor can I assign database index instead of call select() function?

### ISSUE THREAD

**Issue by victorxtx** [phpredis/phpredis#2563]
### Expected behaviour
I tried to assign database index like below:
$redis = new Redis([
  'host' => REDIS_HOST,
  'auth' => REDIS_AUTH,
  'db' => 3
]);
Then, I checked the doc of constructor, it acted as it is...

### Actual behaviour
Sure enought, __construct(): Skip unknown option 'db'
and I have to call: 
$redis->select(3); -- And of course it worked.

-- I'm lookking forward if this option can be added to the constructor.
-- Thank you all.

### I'm seeing this behaviour on
- OS: ubuntu 22.04
- Redis: 7.2.0
- PHP: 8.3.11
- phpredis: 6.0.2

### Steps to reproduce, backtrace or example script
Just like part I.

### I've checked
- [x] There is no similar issue from other users
- [x] Issue isn't fixed in `develop` branch

**Comment by michael-grunder** [comment]
I don't think you can choose the db in the connection array, although I suppose it could be added.

**Comment by victorxtx** [comment]
> I don't think you can choose the db in the connection array, although I suppose it could be added.

So, can it be taken into account?

**Comment by marcing** [comment]
I wonder if ->select() invokes connect() ? In this is the case then lazy connection happens immediatelly for most cases.

**Comment by michael-grunder** [comment]
Support for this was added in 4f6a3ed1

Instead of `db` though you use `database`

```php
$r = new Redis(['host' => 'localhost', 'port' => 6379, 'database' => 3]);
$r->set('foo', 'This is database 3');
```

**Comment by marcing** [comment]
Thank you @michael-grunder !

**Comment by victorxtx** [comment]
> Support for this was added in [4f6a3ed](https://github.com/phpredis/phpredis/commit/4f6a3ed1e71c70f80b631a9f53749e6a9fdb457a)
> 
> Instead of `db` though you use `database`
> 
> $r = new Redis(['host' => 'localhost', 'port' => 6379, 'database' => 3]);
> $r->set('foo', 'This is database 3');

Oh, You Are My Godess! Thank you so much!
